apps/api/src/app/invites/dtos/bulk-invite-members.dto.ts

```diff
@@ -2,7 +2,7 @@ import { IBulkInviteRequestDto } from '@novu/shared';
 import { Type } from 'class-transformer';
 import { ArrayNotEmpty, IsArray, IsDefined, IsEmail, IsNotEmpty, ValidateNested } from 'class-validator';
 
-class EmailInvitee {
+export class EmailInvitee {
```

Comment:
We need to export all classes referenced by Swagger so it can compose the OpenAPI Spec (OAS) successfully.